src/metabase/setup.clj

```diff
@@ -36,6 +37,12 @@
       (t2/select-one-fn :value Setting :key "setup-token")
       (setting/set-value-of-type! :string :setup-token (str (UUID/randomUUID)))))
 
+(defenterprise ignore-internal-user-clause
+  "Returns the where clause to ignore internal metabase user, or an empty where clause."
+  metabase-enterprise.internal-user
+  []
+  [])
```

Comment:
I think this should maybe still filter out the internal user ID on OSS. Imagine an instance that originally starts up the EE version and then switches to OSS. The internal user would have been created but no other users would be, so we'd want to make sure it's filtered out for `has-user-setup`

Reply:
I guess you'd have to move some stuff over to OSS namespaces then. Not sure how best to organize that.

Reply:
I think just putting the internal-user-id into an oss namespace they can both require will work fine here. Good catch